.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,98 @@
+name: Docker
+
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+on:
+  schedule:
+    - cron: '33 5 * * *'
```

Comment:
Can you explain why there's a cron schedule for this?

Reply:
For it to rebuild weekly so it stays up to date with the php:8.3-fpm-alpine docker image.
We don't have to do this for wings as it runs on distroless.

Reply:
I'd rather it only build when we publish a tag/release instead.

Reply:
> I'd rather it only build when we publish a tag/release instead.

Updated my review to not include cron